docs/esa/xmm_newton/xmm_newton.rst

```diff
@@ -85,79 +88,79 @@ This will download the EPIC postcard for the observation '0505720401' and it wil
+     46.71747372703565 -15.78639    12101 ...      Denis-J1228         true
+   -15.881772371450268 -77.61555    33986 ...         Cha-Ha-3         true
 
-This will execute an ADQL query to download the first 10 observations in the XMM-Newton Science Archive. The result of the query will be 
+This will execute an ADQL query to download the first 10 observations in the XMM-Newton Science Archive. The result of the query will be
 stored in the file 'results10.csv'. The result of this query can be printed by doing print(result).
-            
+
 -----------------------------------
 5. Getting table details of XSA TAP
 -----------------------------------
 
-.. code-block:: python
+.. doctest-remote-data::
 
-  >>> from astroquery.esa.xmm_newton import XMMNewton
-  >>>
-  >>> XMMNewton.get_tables()
+  >>> from astroquery.esa.xmm_newton import XMMNewton    # doctest: +IGNORE_OUTPUT
+  >>> XMMNewton.get_tables()  # doctest: +IGNORE_OUTPUT
   INFO: Retrieving tables... [astroquery.utils.tap.core]
   INFO: Parsing tables... [astroquery.utils.tap.core]
   INFO: Done. [astroquery.utils.tap.core]
-  ['tap_schema.columns', 'tap_schema.key_columns', 'tap_schema.keys', 'tap_schema.schemas', 
-  'tap_schema.tables', 'xsa.dual', 'xsa.v_all_observations', 'xsa.v_epic_source', 
-  'xsa.v_epic_source_cat', 'xsa.v_epic_xmm_stack_cat', 'xsa.v_exposure', 'xsa.v_instrument_mode', 
-  'xsa.v_om_source', 'xsa.v_om_source_cat', 'xsa.v_proposal', 'xsa.v_proposal_observation_info', 
-  'xsa.v_publication', 'xsa.v_publication_observation', 'xsa.v_publication_slew_observation', 
-  'xsa.v_public_observations', 'xsa.v_rgs_source', 'xsa.v_slew_exposure', 'xsa.v_slew_observation', 
-  'xsa.v_slew_source', 'xsa.v_slew_source_cat', 'xsa.v_target_type', 'xsa.v_uls_exposure_image', 
+  ['tap_schema.columns', 'tap_schema.key_columns', 'tap_schema.keys', 'tap_schema.schemas',
+  'tap_schema.tables', 'xsa.dual', 'xsa.v_all_observations', 'xsa.v_epic_source',
+  'xsa.v_epic_source_cat', 'xsa.v_epic_xmm_stack_cat', 'xsa.v_exposure', 'xsa.v_instrument_mode',
+  'xsa.v_om_source', 'xsa.v_om_source_cat', 'xsa.v_proposal', 'xsa.v_proposal_observation_info',
+  'xsa.v_publication', 'xsa.v_publication_observation', 'xsa.v_publication_slew_observation',
+  'xsa.v_public_observations', 'xsa.v_rgs_source', 'xsa.v_slew_exposure', 'xsa.v_slew_observation',
+  'xsa.v_slew_source', 'xsa.v_slew_source_cat', 'xsa.v_target_type', 'xsa.v_uls_exposure_image',
   'xsa.v_uls_slew_exposure_image']
 
 This will show the available tables in XSA TAP service in the XMM-Newton Science Archive.
-            
+
 -------------------------------------
 6. Getting columns details of XSA TAP
 -------------------------------------
 
-.. code-block:: python
+.. doctest-remote-data::
 
   >>> from astroquery.esa.xmm_newton import XMMNewton
-  >>>
-  >>> XMMNewton.get_columns('public.v_all_observations')
+  >>> XMMNewton.get_columns('xsa.v_all_observations')  # doctest: +IGNORE_OUTPUT
```

Comment:
same, I would rather keep the column list (my rule of thumb: ignore outputs when it's full of floats, and values that are e.g. a result of a truncated SQL query where it's not guaranteed that the same return will be given every time. But keep strings like these column names in for the tests)